sql/src/main/java/org/opensearch/sql/sql/parser/AstExpressionBuilder.java

```diff
@@ -151,6 +156,30 @@ public UnresolvedExpression visitScalarFunctionCall(ScalarFunctionCallContext ct
     return buildFunction(ctx.scalarFunctionName().getText(), ctx.functionArgs().functionArg());
   }
 
+  @Override
+  public UnresolvedExpression visitNestedFunctionCall(NestedFunctionCallContext ctx) {
+    List<UnresolvedExpression> args = new ArrayList();
+    boolean conditionSet = false;
+
+    if (ctx.nestedFunction().nestedField() != null) {
+      args.add(new QualifiedName(ctx.nestedFunction().nestedField().getText()));
```

Comment:
Don't you need to `visitQualifiedName` here?

Reply:
Fixed in [4e4f4a43d26a3420d401878801154096a6afe728](https://github.com/Bit-Quill/opensearch-project-sql/pull/243/commits/4e4f4a43d26a3420d401878801154096a6afe728)